userguide/object_detection/data-preparation.md

```diff
@@ -9,13 +9,26 @@ toolkit, we will rely on many useful SFrame functions.
 
 The dataset that we will use is [INRIA Annotations for Graz-02
 (IG02)](https://lear.inrialpes.fr/people/marszalek/data/ig02/)[^1][^2]. Our goal is to
-make a detector that can spot bikes and cars. Go ahead and click this link and
-download `ig02-v1.0-bikes.zip` and `ig02-v1.0-cars.zip`. In this tutorial, we
-will assume that you have unzipped these files into `~/Downloads/ig02`. That
-is, you should have the following data structure:
+make a detector that can spot bikes and cars. Open a new terminal window and run the following commands:
+
+```
+$ mkdir -p ~/Downloads/ig02
+$ cd ~/Downloads/ig02
+$ curl https://lear.inrialpes.fr/people/marszalek/data/ig02/ig02-v1.0-bikes.zip > bikes.zip
```

Comment:
I think it's more common to use `curl -O output_name` than piping to a file... but I'm not sure if piping to a file has any drawbacks. This might be fine as is.

Reply:
:+1: This caught my eye too.